packages/components/src/List/ListComposition/Manager/Manager.component.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ListContext } from '../context';
+import useDisplayMode from './useDisplayMode';
+
+export default function Manager(props) {
+	const { initialDisplayMode, onDisplayModeChange } = props;
+	const [displayMode, setDisplayMode] = useDisplayMode(initialDisplayMode, onDisplayModeChange);
+	const contextValues = {
+		collection: props.collection,
+		displayMode: props.displayMode || displayMode,
```

Comment:
Do we share the display mode between multiple nested component in the list ?
It seems more natural to me to override the displaymode hook value directly in the displaymode component:

<List.DisplayMode displayMode='myCustomDisplayMode' />

Reply:
This leads to another question, do we want to use useDisplayMode here or in the displaymode component...
The way its design here can lead to an overwhelming props settings for the manager :
<List.Manager onChangeDisplayMode={() => myStuff} onSortBy={() =mySortBy etc... />

Reply:
I just saw that the display mode value is used in the vlist (make sense :P)
I think we have to talk here, discuss about design and maintenability.

Reply:
Yes, the display mode is shared between VList and DisplayMode selector.
But I think we can manage to do something about it. Let me try to move that to display mode